pygbm/gradient_boosting.py

```diff
@@ -156,18 +156,23 @@ def fit(self, X, y):
             print("Fitting gradient boosted rounds:")
 
         n_samples = X_binned_train.shape[0]
-        # values predicted by the trees. Used as-is in regression, and
-        # transformed into probas and / or classes for classification
+        self.init_predictions_ = self.loss_.get_init_prediction(
```

Comment:
I would rename this to `self.baseline_prediction_` (singular because it does not depend on the number of samples, even if for multiclass it's actually a n-dimensional single prediction :)

Reply:
So `baseline_prediction_` or `singular_prediction_`?

I think I like `baseline` better

Reply:
baseline yes.